src/main/java/com/sprint/mission/discodeit/JavaApplication.java

```diff
@@ -1,428 +1,161 @@
-                    manageChannels();
-                    break;
-                case 3: // 메시지 관리 메뉴 진입
-                    manageMessages();
-                    break;
-                case 4: // 현재 데이터를 파일에 즉시 저장
-                    saveServices();
-                    break;
-                case 5: // 애플리케이션 종료
-                    saveServices(); // 종료 직전에 변경된 데이터 최종 저장
-                    System.out.println("프로그램을 종료합니다. 안녕히 계세요!");
-                    running = false; // 루프 종료하여 프로그램 종료
-                    break;
-                default: // 잘못된 입력 처리
-                    System.out.println("잘못된 선택입니다. 1에서 5 사이의 번호를 입력해주세요.");
-                    break;
-            }
-            System.out.println("\n----------------------------------------\n"); // 메뉴 구분선
-        }
-        sc.close(); // 프로그램 종료, Scanner 자원 해제
-    }
-
-    // --- 서비스 인스턴스 로드 및 저장 메서드 ---
-
-
-    private static void loadServices() {
-        System.out.println("===== 애플리케이션 시작: 데이터 로드 =====");
-        // 각 서비스 구현체 객체를 파일에서 로드 시도. 파일이 없거나 로드 실패 시 새로운 인스턴스 생성.
-        UserRepository UserRepository = persistenceManager.loadUserRepository();
-        if (userRepository == null) {
-            userRepository = new FileUserService();
-            System.out.println("새로운 사용자 서비스 인스턴스 생성.");
-        }
-
-        channelRepository = persistenceManager.loadChannelRepository();
-        if (channelRepository == null) {
-            channelRepository = new FileChannelService();
-            System.out.println("새로운 채널 서비스 인스턴스 생성.");
-        }
-
-        messageRepository = persistenceManager.loadMessageRepository();
-        if (messageRepository == null) {
-            messageRepository = new FileMessageService();
-            System.out.println("새로운 메시지 서비스 인스턴스 생성.");
-        }
-        System.out.println("=========================================\n");
-    }
-
-
-    private static void saveServices() {
-        System.out.println("===== 현재 데이터 저장 시도 중 =====");
-        persistenceManager.saveAllServices(userRepository, channelRepository, messageRepository);
-        System.out.println("===================================\n");
-    }
-
-
-
-    private static void printMainMenu() {
-        System.out.println("====== 채팅 앱 메인 메뉴 ======");
-        System.out.println("1. 사용자 (User) 관리");
-        System.out.println("2. 채널 (Channel) 관리");
-        System.out.println("3. 메시지 (Message) 관리");
-        System.out.println("4. 데이터 저장 (Save Now)"); // 현재 상태를 즉시 파일에 저장하는 메뉴
-        System.out.println("5. 종료");
-        System.out.print("메뉴를 선택하세요: ");
-    }
-
-
-    private static int getUserChoice() {
-        while (!sc.hasNextInt()) { // 정수가 입력될 때까지 반복
-            System.out.println("숫자를 입력해주세요.");
-            sc.next(); // 잘못된 입력(토큰) 소비하여 무한 루프 방지
-            System.out.print("메뉴를 다시 선택하세요: ");
+        System.out.println("--- 테스트 시작 ---");
+
+        //리포지토리 인스턴스 생성
+        JCFUserRepository userRepository = new JCFUserRepository();
```

Comment:
구현체보단 인터페이스로 선언하는게 나중에 확장하기에 용이해요.  
```suggestion
        UserRepository userRepository = new JCFUserRepository();
```